.gitattributes

```diff
@@ -2,10 +2,11 @@
 # https://www.kernel.org/pub/software/scm/git/docs/gitattributes.html
 
 # Ignore all test and documentation with "export-ignore".
-/.gitattributes     export-ignore
-/.gitignore         export-ignore
-/.travis.yml        export-ignore
-/phpunit.xml.dist   export-ignore
-/.scrutinizer.yml   export-ignore
-/tests              export-ignore
-/.editorconfig      export-ignore
+/.gitattributes         export-ignore
+/.gitignore             export-ignore
+/.travis.yml            export-ignore
+/phpunit.xml.dist       export-ignore
+/.scrutinizer.yml       export-ignore
+/tests                  export-ignore
+/.editorconfig          export-ignore
+/.phpunit.result.cache  export-ignore
```

Comment:
It's an (git) ignored file, why should it be listed here?

Reply:
My bad 😅 

it's really not necessary, I'll fix it.

Reply:
Removed!